src/sql/src/plan/func.rs

```diff
@@ -902,6 +902,13 @@ lazy_static! {
                     Ok(ScalarExpr::literal_null(ScalarType::String))
                 })
             },
+            "pg_encoding_to_char" => Scalar {
+                // Materialize only supports UT8-encoded databases. Return 'UTF8' if Postgres'
+                // encoding id for UTF8 (6) is provided, otherwise return 'NULL'.
+                params!(Int64) => sql_op!(
+                    "(SELECT CASE WHEN $1 = 6 THEN 'UTF8'::text ELSE 'NULL'::text END pg_encoding_to_char)"
```

Comment:
This returns the literal string 'NULL', rather than an actual `NULL` value. I believe you just want `NULL` to match PostgreSQL's behavior on unknown inputs.

Also, no need for either the subquery or the column alias—the column alias returned here is ignored, anyway. I _think_ just the following should work?

```
sql_op!("CASE WHEN $1 = 6 THEN 'UTF8' ELSE NULL END")
```